backend/src/test/java/de/bund/digitalservice/ris/caselaw/domain/LookupTableImporterServiceTest.java

```diff
@@ -49,9 +47,12 @@ class LookupTableImporterServiceTest {
 
   @SpyBean private LookupTableImporterService service;
 
-  @MockBean private DatabaseDocumentTypeRepository databaseDocumentTypeRepository;
+  @MockBean
+  private de.bund.digitalservice.ris.caselaw.adapter.database.r2dbc.lookuptable
+          .DatabaseDocumentTypeRepository
+      databaseDocumentTypeRepository;
 
-  @MockBean private JPADocumentTypeRepository jpaDocumentTypeRepository;
+  @MockBean private JPADocumentTypeRepository JPADocumentTypeRepository;
```

Comment:
Variable/field names start with a lowercase letter. In this case I'm total fine with the old version. or jPADocumentTypeRepository

Reply:
Solved 👍